lib/response.js

```diff
@@ -529,6 +529,19 @@ function patch(Response) {
                 }
                 return flush(self, formattedBody);
             }
+            // Preserve original error when
+            // Content-type is set, _strictFormatters=false, no formatter found
+            if (body instanceof Error) {
```

Comment:
Couldn’t this be something other than an error? Maybe check against what we assert in `flush()`?

Reply:
I don't understand @DonutEspresso, isn't this PR specifically for preserving errors?

Reply:
It's been awhile since I looked at this - I _think_ my question was if it was possible to get to this point in the function where body isn't an Error but just a POJO. IOW, does this new `if` clause cause a silent failure/hang where there's no formatter, and the body is not an Error?

Reply:
Maybe there's already a test for this in which case carry on!